packages/badge/style.module.css

```diff
@@ -30,7 +30,7 @@
     --color: var(--white);
   }
   &.secondary {
-    --background-color: var(--gray-6);
+    --background-color: var(--vagrant-secondary);
```

Comment:
Saw your comment in Figma, but no go on the `brand-secondary` color?

Reply:
@mwickett Sounds like a temporary solution until we can revisit colors more holistically.